File: JAVA/leapyear.java
import java.util.Scanner;

public class leapyear {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter a Year :");
        int year = sc.nextInt();

        if (year % 400==0) {
            System.out.println("It Was a leap year ");
            
        }
        else if (year % 4 == 0 && year % 100 != 0) {

            System.out.println("IT Was a Leap Year ");
            
            
        }

        else {
            System.out.println("IT Was Not A  leap year ");
        }
    }
}

File: BY-PYTHON/leapyear.py
n = int(input("Enter  a  year  : "))

if  n % 400 == 0:
    print("It is a leap Year ")
    
elif n % 100 ==0 and n%100 != 0:
    print("It is a leap Year")
    
else:
    print("IT Is NOt A Leap Year ")